internal/client/model.go

```diff
@@ -193,6 +193,13 @@ type UniversalMachineIdentityAuthResponse struct {
 	TokenType         string `json:"tokenType"`
 }
 
+type OidcMachineIdentityAuthResponse struct {
```

Comment:
I think we can just combine `UniversalMachineIdentityAuthResponse` and `OidcMachineIdentityAuthResponse` into a single struct called `MachineIdentityAuthResponse`

Reply:
yup I agree